packages/pilot/src/pages/Recipients/Search/Search.js

```diff
@@ -176,26 +175,57 @@ class RecipientsSearch extends React.Component {
 
   requestData (query) {
     this.props.onRequestSearch({ query })
-    const searchQuery = {}
-    let key = 'name'
-
-    if (query && query.search) {
-      if (isRecipientId(query.search)) {
-        key = 'id'
-      } else if (isBanckAccount(query.search)) {
-        key = 'bank_account_id'
+
+    const findByQuery = (search) => {
+      let key = 'name'
+
+      if (search && search.search) {
+        if (isRecipientId(search.search)) {
+          key = 'id'
+        } else if (isBankAccount(search.search)) {
+          key = 'bank_account_id'
+        }
       }
 
-      searchQuery[key] = query.search
+      return this.props.client
+        .recipients
+        .find({
+          [key]: query.search,
+          count: query.count,
+          page: query.offset,
+        })
+        .then((recipients) => {
+          if (Array.isArray(recipients)) return recipients
```

Comment:
Aqui daria pra simplificar essa verificação com um flatten, não? Como: `flatten([ recipients ])`

Reply:
Também acho que fica mais conciso com `flatten`, mas gosto bem mais da forma atual, que deixa mais clara a lógica para quem estiver lendo.

Reply:
Gostei da sugestão, mas no caso acho que poderia ser feito diferente, que tal assim:

```suggestion
     .then(recipients => [recipients])
```
No caso seria retornado `[conteudo]` ou `[[conteudo]]`, mas como em baixo já tem um `flatten` que é suficiente para manter a mesma interface.